changelog/_unreleased/2025-03-20-move-notifications-from-admin-to-core.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Move notifications from admin to core
+---
+# Core
+* Added `Shopware\Core\Framework\Notification\NotificationCollection`, `Shopware\Core\Framework\Notification\NotificationDefinition` & `Shopware\Core\Framework\Notification\NotificationEntity`
+___
+# Administration
+* Deprecated `Shopware\Administration\Notification\NotificationCollection`, `Shopware\Administration\Notification\NotificationDefinition` & `Shopware\Administration\Notification\NotificationEntity`
+___
+# Upgrade Information
+## Deprecated admin notification entity + related classes
+
+We have moved the notification entity, collection and definition to core. You should update your code to reference the new classes:
+
+* `Shopware\Core\Framework\Notification\NotificationCollection`
+* `Shopware\Core\Framework\Notification\NotificationDefinition`
+* `Shopware\Core\Framework\Notification\NotificationEntity`
+___
+# Next Major Version Changes
```

Comment:
should we leave this out, and directly put the content to the 6.7 and 6.8 upgrade guide?

Reply:
So I would put the `Next Major Version Changes` in 6.8 and `Upgrade Information` in 6.7 upgrade guide?

Reply:
I am not sure, if we already do this @keulinho @alvarothomas

Reply:
i would put it directly into upgrade guide, makes it already easier